_static/style.css

```diff
@@ -1,3 +1,15 @@
 .bd-header-announcement {
   background-color: var(--pst-color-info-bg);
 }
+
+.sphinx-codeautolink-a:link {
+  border-bottom-color: lightgray;
+  border-bottom-style: dotted;
+  border-bottom-width: 1px;
+}
+
+.sphinx-codeautolink-a:hover {
+  border-bottom-color: rgb(255, 139, 139);
+  border-bottom-style: dotted;
+  border-bottom-width: 1px;
+}
```

Comment:
@dcherian @zmoon I think a subtle dotted underline is nice, personally I think the thick underlines (https://scikit-image.org/docs/stable/auto_examples/segmentation/plot_regionprops.html) are a bit distracting and distinct from the jupyterlab experience, but open to other opinions!

Reply:
I concur!

Reply:
A little darker would be nicer to me. It's also convention that solid lines are clickable links, so perhaps just a thin darker line?